<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CastedVotes</class>
 <widget class="QDialog" name="CastedVotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2980b9;</string>
  </property>
  <widget class="QPushButton" name="Header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid white;
                    padding: 0 8px;
                    background-color: green;
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string> LBTCSU VOTING  SYSTEM</string>
   </property>
  </widget>
  <widget class="QFrame" name="BackgroundFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>1311</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
    background-color: rgb(0, 54, 79); 
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;
    color: white;
}
QTableView {
    background-color: #1B263B;
    color: #FFFFFF;
    selection-background-color: #1B98E0;
    gridline-color: #415A77;
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;
}
QPushButton {
    background-color: #1B98E0;
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
}
QLineEdit {
    background-color: #2E3A59;
    color: white;
	border-radius: 7px;
	border: 1 solid blue;
}
QComboBox {
	background-color: #2E3A59;
    color: white;
	border-radius: 7px;
	border: 1 solid blue;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_search_Voter">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_Search">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QLineEdit{
                    color: rgb(255, 255, 255);
                    border:2px solid white;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Enter Your Student ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Position">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QComboBox{
                    color: rgb(255, 255, 255);
                    border:2px solid white;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <item>
     <property name="text">
      <string>President</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vice President</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>General Secretary</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finance Coordinator</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sports Coordinator</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_Reset">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
    </property>
   </widget>
   <widget class="QFrame" name="VoterDetailFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2980b9;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="image_field">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>4</y>
       <width>161</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">  
 QLabel {
  border:2px solid green;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
 
};</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>129</y>
       <width>451</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPlainTextEdit{

    background-color: #1B263B;
    color: #FFFFFF;
    selection-background-color: #1B98E0;
    gridline-color: #415A77;
    border-top-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-left-radius: 12px;
    border-bottom-right-radius: 12px;


}</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_search_Candidet">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>20</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>161</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  
 QLabel {
  border:2px solid green;
  border-radius: 10px;
  padding: 8px 8px;
  background: rgb(255, 255, 255);
 
};</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="SubHeading">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>641</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">     
QPushButton{
                    color: rgb(255, 255, 255);
                    border:1px solid green;
                    border-radius: 6px;
                    padding: 0 8px;
                    background-color: green;
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>SEARCH YOUR RECORD THEN SEARCH CANDIDATE RECORD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>50</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    
QPushButton{
                    color: rgb(255, 255, 255);
                    border:2px solid green;
                    border-radius: 8px;
                    padding: 0 8px;
                    background-color: rgb(255, 267, 245);
                    font: 500 11pt ;
                   

}

 </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::GoHome"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
